website/docs/events/m1.md

```diff
@@ -40,7 +42,125 @@ Please consider topics such as:
 
 During the workshop, participants will put their ideas into practice and take the first steps towards having a full-fledged class that uses modern technology for efficient teaching.
 
-## Open Discussion over Lunch: 13:00 - 14:00
+### Open Discussion over Lunch: 13:00 - 14:00
 
 Lunch is on the Open Education Hub team.
 As always, lunch will be a pretext for more open-ended discussions regarding the Open Education Hub project and the guests' further involvement with it. 
+
+## Summary
+
+Find the entire discussion from our meeting below in text format or on [YouTube](https://www.youtube.com/watch?v=RkUnCXL_BtA&ab_channel=OpenEducationHub) in audio format (Romanian only).
+
+### Project Overview
+
+RD: We want to see how realistic is our idea.
```

Comment:
- Provide explanations for who is who (RD, RN etc.)
- One sentence per line
- Leave empty lines between interventions. Now this looks like a giant blob of text.

Reply:
On it.